mmcv/runner/base_runner.py

```diff
@@ -354,8 +353,8 @@ def resume(self,
                 self.logger.info('the iteration number is changed due to '
                                  'change of GPU number')
 
-        if resume_meta:
-            self.meta = checkpoint['meta']
+        # resume meta information meta
+        self.meta = checkpoint['meta']
```

Comment:
what if checkpoint does not have 'meta'? in case of BC-breaking, shall we use `checkpoint.get('meta', None)`?

Reply:
That's also my original concern. But after reading the codes, I found the original resume function already assumes checkpoint dict always has 'meta' inside (https://github.com/open-mmlab/mmcv/blob/0e01dfc7701821e3f5b8c92fac28a3da7aaf6d62/mmcv/runner/base_runner.py#L340). It makes sense because the checkpoint for resuming is generated by mmcv, hence it must contain 'meta' in the checkpoint file. (And there's no BC breaking here)

Reply:
OK, so we do not need to do that..

Reply:
bug here. The new config in the resuming run (can be different from the one loaded from meta) is overwritten.